src/ert/scheduler/lsf_driver.py

```diff
@@ -414,7 +414,7 @@ async def kill(self, iens: int) -> None:
                 return_on_msgs=(JOB_ALREADY_FINISHED_BKILL_MSG),
             )
             await asyncio.create_subprocess_shell(
-                f"sleep {self._sleep_time_between_bkills}; {self._bkill_cmd} -s SIGKILL {job_id}",
+                f"nohup sh -c 'sleep {self._sleep_time_between_bkills}; {self._bkill_cmd} -s SIGKILL {job_id}' &",
```

Comment:
I thought this would work as we already have `start_new_session=True`. Can we remove that flag with nohup?

Reply:
As long as  `start_new_session` works (I think it does until proven otherwise), `start_new_session` is the approach to go for.

Reply:
Haven't got to test it yet. If it works I'll close the PR.